model/minsky.cc

```diff
@@ -267,7 +267,10 @@ namespace minsky
     canvas.model->addGroup(g);
     m.populateGroup(*g);
     // Default pasting no longer occurs as grouped items or as a group within a group. Fix for tickets 1080/1098    
-    canvas.selection.clear();    
+    canvas.selection.clear();        
+
+    // Needed to ensure all wires are copied in a complex selection as in, for example, 1Free.mky. For ticket 1190
+    g->splitBoundaryCrossingWires();    
```

Comment:
SplitBoundaryCrossingWires is only for groups. Only needed when items are moved between groups.

Reply:
With copy/paste, wires should not be in the selection unless both endpoints are in the selection. In which case, the wires will not be crossing group boundaries.

Reply:
I understand now. The paste operation sometimes fails with wires that are very curved, at least in the 1Free.mky example that has been my experience. It has something to do with groups.